README.md

```diff
@@ -225,15 +238,17 @@ render({fonts}) {
       models: fonts,
       $template: 'font_t',
       decorator: `decorator`,
```

Comment:
Should we update this (and elsewhere, to be consistent) to `myDecorator` like you did above? Or move the edit to `myDecorator` back to just `decorator`? Either seems ok, but consistency would be good. I get why you probably edited it to `myDecorator, to be clearer/less confusing.